src/GEMItem.cpp

```diff
@@ -234,6 +234,89 @@ GEMItem::GEMItem(char* title_, double& linkedVariable_, boolean readonly_)
 
 //---
 
+GEMItem::GEMItem(char* title_, byte& linkedVariable_, GEMPage* linkedPage_)
+  : title(title_)
+  , linkedVariable(&linkedVariable_)
+  , linkedPage(linkedPage_)
+  , linkedType(GEM_VAL_BYTE)
+  , readonly(false)
+  , type(GEM_ITEM_LINKED_VAL)
+{ }
+
+GEMItem::GEMItem(char* title_, byte& linkedVariable_, GEMPage& linkedPage_)
+  : GEMItem(title_, linkedVariable_, &linkedPage_) // call GEMPage* constructor
```

Comment:
wirklich `GEMPage` constructor? Meinst du nicht:
```suggestion
  : GEMItem(title_, linkedVariable_, &linkedPage_) // call GEMItem* constructor
```
und wiesso `*`?

Reply:
Wir machen aus der `GEMPage` Refrenez einen Pointer (mit dem `&linkedPage`), da die intern sowieso nur in eine Pointer abgespeichert wird und rufen damit den GEMItem Konstruktor auf, welcher einen `GEMPage*` erwartet.

Reply:
ok aber dann ist meine suggestion korrekt oder nicht? wir rufen den `GEMItem` constructor auf und nicht den `GEMPage` constructor.